test/Common.cpp

```diff
@@ -166,28 +163,19 @@ bool CommonOptions::parse(int argc, char const* const* argv)
 
 	if (vmPaths.empty())
 	{
-		std::string evmone = envOrDefaultPath("ETH_EVMONE", evmoneFilename);
-		if (!evmone.empty())
-			vmPaths.emplace_back(evmone);
+		if (auto envPath = getenv("ETH_EVMONE"))
+			vmPaths.emplace_back(envPath);
+		else if (auto repoPath = findInDefaultPath(evmoneFilename))
+			vmPaths.emplace_back(*repoPath);
 		else
-		{
-			std::cout << "Unable to find " << solidity::test::evmoneFilename
-				 << ". Please provide the path using --vm <path>." << std::endl;
-			std::cout << "You can download it at" << std::endl;
-			std::cout << solidity::test::evmoneDownloadLink << std::endl;
-		}
-	}
-
-	if (ewasm) {
-		std::string hera = envOrDefaultPath("ETH_HERA", heraFilename);
-		if (!hera.empty())
-			vmPaths.emplace_back(hera);
-		else {
-			std::cout << "Unable to find " << solidity::test::heraFilename
-					  << ". Please provide the path using --vm <path>." << std::endl;
-			std::cout << "You can download it at" << std::endl;
-			std::cout << solidity::test::heraDownloadLink << std::endl;
-			std::cout << "Ewasm tests disabled." << std::endl;
+			vmPaths.emplace_back(evmoneFilename);
+		if (ewasm) {
```

Comment:
Note that I moved this into the ``if (vmPaths.empty())`` now, slightly changing the behaviour, but ending up more reasonable I would say.
If you specify the vm paths explicitly via arguments and want ewasm, you need to specify both vms.